gnuplot.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Make sure gnuplot is installed.
+if ! [ -x "$(command -v gnuplot)" ]; then
+    echo "Please install gnuplot"
```

Comment:
Me neither, good idea.

Reply:
I've decided to use `command not found: gnuplot`, since that should be familiar to most users due to shell scripts and `gnuplot` might be installed just not in the `PATH`.